lib/rosette/integrations/smartling-integration/smartling_puller.rb

```diff
@@ -1,135 +1,217 @@
 # encoding: UTF-8
 
+require 'set'
+require 'concurrent'
+
 module Rosette
   module Integrations
     class SmartlingIntegration < Integration
       class SmartlingPuller
 
-        attr_reader :rosette_config, :smartling_apis, :completed_files_map, :locale
+        DEFAULT_THREAD_POOL_SIZE = 15
```

Comment:
It seems odd to have two different settings unless they are VERY different,
but thats just me thinking as a consumer of it
On Feb 11, 2015 8:21 AM, "Cameron Dutro" <notifications@github.com> wrote:

> In lib/rosette/integrations/smartling-integration/smartling_puller.rb
> <https://github.com/rosette-proj/rosette-integration-smartling/pull/1#discussion_r24507955>
> :
>
> >  module Rosette
> >    module Integrations
> >      class SmartlingIntegration < Integration
> >        class SmartlingPuller
> >
> > -        attr_reader :rosette_config, :smartling_apis, :completed_files_map, :locale
> > +        DEFAULT_THREAD_POOL_SIZE = 15
>
> Ya they're different operations. I think we cap our MySQL connections to
> 16, so I chose 15 to leave some wiggle room. I think however it might be
> better to use 10 here too to leave more than just a single free connection.
> What do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rosette-proj/rosette-integration-smartling/pull/1/files#r24507955>
> .
>